BT.json

```diff
@@ -505,7 +505,7 @@
 			"wowID":"31101"
 		},
 		"Pauldrons of the Forgotten Protector":{
-			"prio":["Hunter/DPS War > Ele/Resto sham","EP/GP"],
+			"prio":["Chiron (1)","Dillmcpickle (1)","Thotpatrol (1)","Ferbb (1)","Oglol (2)","Putitah (2)","Paintrain (2)","Hunter/DPS War > Ele/Resto sham","EP/GP"],
```

Comment:
Prot warrior first, the DPS shoulders are bis for threat while trading 10 stam for 100+ additional armor.

Reply:
if you feel strongly about shoulders we can get your warrior an earlier shoulder token, but i dont think its a big enough of an upgrade of T5 prot shoulders to not give to the dps when its one of the biggest pieces hunters/warriors get all tier. ex rushing hunters on 2pc is massive, same with warriors for overall dmg. theres so much tanking gear for free for the prot warr, and if you need a tank with t6 theres a druid and pally that can MT basically every fight. and they will have their 4/5 set as soon as it drops.

Reply:
Spoke offline, let's take a look after progression and see if threat is an issue.  I'm fine with this for now.